fedimint-server/src/lib.rs

```diff
@@ -265,20 +263,15 @@ impl FedimintServer {
                             target: LOG_NET_API,
                             path, "API handler panicked, DO NOT IGNORE, FIX IT!!!"
                         );
-                        jsonrpsee::core::Error::Call(CallError::Custom(ErrorObject::owned(
-                            500,
-                            "API handler panicked",
-                            None::<()>,
-                        )))
+                        ErrorObject::owned(500, "API handler panicked", None::<()>)
                     })?
                     .map_err(|tokio::time::error::Elapsed { .. }| {
-                        jsonrpsee::core::Error::RequestTimeout
+                        // TODO: find a better error for this, the error we used before:
+                        // jsonrpsee::core::Error::RequestTimeout
+                        // was moved to be client-side only
+                        ErrorObject::owned(-32000, "Request timeout", None::<()>)
```

Comment:
so this is the -32000 we were hitting before

Reply:
I actually don't know what's the right code for request timeout, but jsonrpc error codes are not http codes and are like this.

Reply:
yea we should define our own error codes, -32000 is "unknown error code"